#! /usr/bin/env python
# Task Long configuration

PACKAGE='floor_nav'
import roslib; roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *
from task_manager_lib.parameter_generator import *

gen = TaskParameterGenerator()
#       Name                    Type            Description
#       Default    Min    Max
gen.add("camera_width",     double_t, 0,    "horizontal resolution of the camera in pixels",  1280)
gen.add("k_theta",        double_t, 0,    "Gain for angular control of the robot angle",  3.0)
gen.add("max_angular_velocity",   double_t, 0,    "Max allowed velocity",  5.0)
gen.add("angle_threshold",double_t, 0,    "Angle at which the target is considered reached",  0.1)


exit(gen.generate(PACKAGE, "floor_nav", "TaskStareAtFace"))

